### PR TITLE
Proteomes

### PR DESCRIPTION
**Pull request by 4ndr01d3**

### CODE REVIEW

File: src/components/Entry/DomainArchitectures/index.js [hunk 1]
```diff
@@ -167,37 +175,39 @@ class DomainArchitectures extends PureComponent {
...
+                      }}
+                    >
+                      {' '}
+                      {d}{' '}
+                    </Link>{' '}
+                    -{' '}
                   </span>
                 ))}
-                <IDAProtVista matches={idaObj.domains} length={idaObj.length} />
+                <IDAProtVista matches={idaObj.domains} length={1000} />
```
Comment (aurel-l) [inline src/components/Entry/DomainArchitectures/index.js]: Why was that changed to a fixed value, is that temporary?

Comment (4ndr01d3) [reply src/components/Entry/DomainArchitectures/index.js]: Yes, it is now constant because the domains don't have length, so everything is now calculated on an hypothetical length of 1000, however I should've  use the previously defined constant length. (line 61)

Comment (aurel-l) [reply src/components/Entry/DomainArchitectures/index.js]: Ah, OK, makes sense. Do you want to do the change to use a common const value?

Comment (4ndr01d3) [reply src/components/Entry/DomainArchitectures/index.js]: Done!
